Update WebM support in glossary page (#33686)
mdn/content: files/en-us/glossary/webm/index.md

{{GlossarySidebar}}

> **WebM** is royalty-free and is an open web video format natively supported in all modern browsers.

Reviewer: Linking to more detailed docs on MDN.

```suggestion
**[WebM](/en-US/docs/Web/Media/Formats/Containers#webm)** is royalty-free and is an open web video format natively supported in all modern browsers.
```

> **[WebM](/en-US/docs/Web/Media/Formats/Containers#webm)** is royalty-free and is an open web video format natively supported in all modern browsers.

Reviewer: No, you're right.

```suggestion
**[WebM](/en-US/docs/Web/Media/Formats/Containers#webm)** is a royalty-free open web video format natively supported in all modern browsers.
```

> **[WebM](/en-US/docs/Web/Media/Formats/Containers#webm)** is a royalty-free open web video format that is natively supported in all modern browsers.

Author: Sorry, more nitpicking. Is "royalty-free" significant here / different from "open"?

Author: What about "WebM is an open, royalty-free web video format..."

Reviewer: Yes, as I understand it royalty free and open imply different things. You can have an open format, where access to the source code and/or specification is open, but you're still expected to pay royalties in a commercial setting.

The fact that it is royalty free is actually the more important thing for developers. Being open helps with debugging etc, but not having to track use and pay for it is what makes the format successful.

So I would prefer the current ordering - unless you have a strong counter-argument for open being the important thing here?

Author: I understand now, thanks! Just going to add a [comma between the the two adjectives](https://www.thepunctuationguide.com/comma.html#:~:text=Multiple%20adjectives,same%20noun)

> **[WebM](/en-US/docs/Web/Media/Formats/Containers#webm)** is a royalty-free, open web video format that is natively supported in all modern browsers.

Reviewer: I don't know why this grates, but it does. How about:
```suggestion
**[WebM](/en-US/docs/Web/Media/Formats/Containers#webm)** is a royalty-free and open web video format, which is natively supported in all modern browsers.
```
